host-spec/c04-networking.tm

```diff
@@ -81,551 +128,427 @@
-
-    \ \ \ \ // to the child type storage location.
-
-    \ \ \ \ bytes storage_key = 3;
-
-    \ \ \ \ // Storage keys.
-
-    \ \ \ \ repeated bytes keys = 6;
-
-    }
-
-    \;
-  </verbatim>
-
-  <subsubsection|RemoteHeaderRequest>
-
-  Remote header request.
-
-  <\verbatim>
-    \;
-
-    message RemoteHeaderRequest {
-
-    \ \ \ \ // Block number to request header for.
-
-    \ \ \ \ bytes block = 2;
-
-    }
-
-    \;
-  </verbatim>
-
-  <subsubsection|RemoteHeaderResponse>
-
-  Remote header response.
-
-  <\verbatim>
-    \;
-
-    message RemoteHeaderResponse {
-
-    \ \ \ \ // Header. None if proof generation has failed
-
-    \ \ \ \ // (e.g. header is unknown).
-
-    \ \ \ \ bytes header = 2; // optional
-
-    \ \ \ \ // Header proof.
-
-    \ \ \ \ bytes proof = 3;
-
-    }
-
-    \;
-  </verbatim>
-
-  <subsubsection|RemoteChangesRequest>
-
-  Remote changes request.
-
-  <\verbatim>
-    \;
-
-    message RemoteChangesRequest {
-
-    \ \ \ \ // Hash of the first block of the range (including first)
-
-    \ \ \ \ // where changes are requested.
-
-    \ \ \ \ bytes first = 2;
-
-    \ \ \ \ // Hash of the last block of the range (including last)
-
-    \ \ \ \ // where changes are requested.
-
-    \ \ \ \ bytes last = 3;
-
-    \ \ \ \ // Hash of the first block for which the requester has
-
-    \ \ \ \ // the changes trie root. All other
-
-    \ \ \ \ // affected roots must be proved.
-
-    \ \ \ \ bytes min = 4;
-
-    \ \ \ \ // Hash of the last block that we can use when
-
-    \ \ \ \ // querying changes.
-
-    \ \ \ \ bytes max = 5;
-
-    \ \ \ \ // Storage child node key which changes are requested.
-
-    \ \ \ \ bytes storage_key = 6; // optional
-
-    \ \ \ \ // Storage key which changes are requested.
-
-    \ \ \ \ bytes key = 7;
-
-    }
-
-    \;
-  </verbatim>
-
-  <subsubsection|RemoteChangesResponse>
-
-  Remote changes response.
-
-  <\verbatim>
-    \;
-
-    message RemoteChangesResponse {
-
-    \ \ \ \ // Proof has been generated using block with this number
-
-    \ \ \ \ // as a max block. Should be less than or equal to the
-
-    \ \ \ \ // RemoteChangesRequest::max block number.
-
-    \ \ \ \ bytes max = 2;
-
-    \ \ \ \ // Changes proof.
-
-    \ \ \ \ repeated bytes proof = 3;
-
-    \ \ \ \ // Changes tries roots missing on the requester' node.
-
-    \ \ \ \ repeated Pair roots = 4;
-
-    \ \ \ \ // Missing changes tries roots proof.
-
-    \ \ \ \ bytes roots_proof = 5;
-
-    }
-
-    \;
-
-    // A pair of arbitrary bytes.
-
-    message Pair {
-
-    \ \ \ \ // The first element of the pair.
-
-    \ \ \ \ bytes fst = 1;
-
-    \ \ \ \ // The second element of the pair.
-
-    \ \ \ \ bytes snd = 2;
-
-    }
-
-    \;
-  </verbatim>
-
-  <subsection|Finality Package>
-
-  ProtoBuf details:
-
-  <\itemize>
-    <item>syntax: proto3
-
-    <item>package: api.v1.finality
+    <item><verbatim|/paritytech/grandpa/1> - a substream/notification
```

Comment:
Name will change in the future: https://github.com/paritytech/substrate/issues/7252